src/components/Tooltip/Tooltip.tsx

```diff
@@ -86,7 +88,7 @@ export const TooltipContent = forwardRef<
     <FloatingPortal>
       <div
         ref={ref}
-        className={className}
+        className={classNames('Layer__tooltip', className)}
```

Comment:
Fyi, `Layer__tooltip` class is added manually to `<Tooltip .../>` component in many places. I'm going refactor this with another PR once this is merged.